build/prepush.sh

```diff
@@ -21,24 +21,20 @@ check_and_commit_updated_translations() {
 # lint, test, and build assets to update translations
 prepush() {
     echo "--------------------------------------------------------"
-    echo "Linting"
-    echo "--------------------------------------------------------"
-    yarn lint || exit 1
```

Comment:
Why lint is unnecessary?

Reply:
We use a plugin called `lint-staged` which operates the way its name suggests, linting all staged files before they are committed.

Reply:
We use a plugin called `lint-staged`, which operates exactly how it sounds. Its automatically lints all staged files before they're committed.

Reply:
We use already `lint-staged` to lint on pre-commit.